dataflux_pytorch/benchmark/checkpointing/multinode/README.md

```diff
@@ -4,56 +4,115 @@ This benchmarking script will allow you to run and benchmark the performance of
+  * `CKPT_DIR_PATH`: The path to the directory towhich the checkpoints will be written.
 
-  * `CKPT_RESTORE_PATH`: The path to restore checkpoints from. Typically the `CKPT_DIR_PATH` + `/checkpoints/`
+  * `CKPT_RESTORE_PATH`: The path to the directory from which the checkpoints will be loaded.
 
-2. Set the optional environment variables, if desired:
+1. The following environment variables are optional. Default values will be used if not set:
   
-  * `NUM_LAYERS`: The number of layers in the model, which affects the size of the model and therefore the size of the checkpoints. Defaults to 10.
+  * `NUM_LAYERS`: The number of layers in the model, which affects the size of the model and therefore the size of the checkpoints. Defaults to `10`.
   
-  * `ACCELERATOR`: Set to `gpu` if running on a GPU, or `cpu` if running on a CPU (default)
-    * If running on a GPU, you also must set `PJRT_DEVICE` to `CPU`.
+  * `ACCELERATOR`: Set to `gpu` if running on a GPU, or `cpu` if running on a CPU. Defaults to `cpu`.
+    * If running on GPU(s) `PJRT_DEVICE` must be set to `CPU`.
   
-  * `NUM_SAVE_CALLS`: The number of times `trainer.save_checkpoint` is called. Defaults to 3.
+  * `NUM_SAVE_CALLS`: The number of times `trainer.save_checkpoint` is called. Defaults to `3`.
   
-  * `NUM_LOAD_CALLS`: The number of times `trainer.strategy.load_checkpoint` is called. Defaults to 3. 
+  * `NUM_LOAD_CALLS`: The number of times `trainer.strategy.load_checkpoint` is called. Defaults to `3`. 
 
-  * `NUM_NODES`: The number of nodes (machines). Defaults to 1.
+  * `NUM_NODES`: The number of nodes you wish to deploy the workload on. Defaults to `1`.
 
-  * `NUM_DEVICES`: The number of devices per node, or `auto`. Defaults to `auto`. 
+  * `NUM_DEVICES`: The number of devices per node. Defaults to `auto`. 
 
+### Local Execution
 
-### Installing Requirements:
- 
-* Install the requirements using following command `pip install -r dataflux_pytorch/benchmark/requirements.txt`; `pip install .`
+#### Dataflux
+```shell
+python3 -m dataflux_pytorch.benchmark.checkpointing.multinode.train --strategy=dataflux_fsdp
+```
 
+#### FSSpec
+```shell
+python3 -m dataflux_pytorch.benchmark.checkpointing.multinode.train --strategy=fsspec_fsdp
+```
 
-### Checkpointing Strategy: 
+#### Boot Disk
+```shell
+# Benchmark checkopint saves first
+python3 -m dataflux_pytorch.benchmark.checkpointing.multinode.train --strategy=fsdp --save_only
 
-* Custom Strategy needs to be implemented and passed to the trainer. The strategy needs to extend [Strategy class](https://lightning.ai/docs/pytorch/stable/api/lightning.pytorch.strategies.Strategy.html) or one of the class which extend Strategy class. When using Strategy which extends [FSDPStrategy](https://lightning.ai/docs/pytorch/stable/api/lightning.pytorch.strategies.FSDPStrategy.html), GPU is required.
+# Benchmark loads next
```

Comment:
what does "loads next" mean here? Loads only?

Reply:
Moved to a new section at the end.